### PR TITLE
Ragmad01 add page

### PR DESCRIPTION
**Pull request by Ragmad01**

### CODE REVIEW

File: _pages/Ragmad01.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+githubHandle: Ragmad01
+pageUrl: https://ragmad01.github.io/Raghavs-repository/
```
Comment (hectorsector) [inline _pages/Ragmad01.yaml]: Great start @Ragmad01! You don't need the `https://` in front of the URL.

**Comment by crichID** [comment]
@Ragmad01 I went ahead and fixed the syntax issues in https://github.com/githubschool/on-demand-github-pages/pull/1337/commits/ad1d5ac2411e028653c01127048f3c6cf6e09992. Please take a quick look at the changes. Once the checks have finished, please feel free to click the big green **Merge pull request** button. If you haven't merged it by August 4th, I will go ahead and do it just to keep the repository clean!

**Comment by Ragmad01** [comment]
ok thanks

**Comment by Ragmad01** [comment]
thanks

**Pull request merged**